### PR TITLE
Build script for swagger-parser

### PR DESCRIPTION
**Pull request by sapnashukla28**
Package: swagger-parser
github url: https://github.com/swagger-api/swagger-parser.git

**Done for  the latest released version** 
- The version given was 1.0.37
- For this version the build was passing, however the tests were failing, 
 FileReferenceTest.testIssue421:208 expected [true] but found [false]
  SwaggerReaderTest.testIssue136:290 NullPointer
- Same test failure were observed for intel as well, Test status is pairity.
- Created build script for latest released version v2.0.28, wherein both build and test is passing.

Thanks,
Sapna.

### CODE REVIEW

File: s/swagger-parser/swagger-parser_ubi8.4.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+export WORK_DIR=$HOME_DIR/$PACKAGE_NAME
+cd $WORK_DIR
+git checkout $PACKAGE_VERSION
+
+function try_mvn(){
+	
+	if ! mvn clean install; then
+		echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+		echo "$PACKAGE_VERSION $PACKAGE_NAME" > $HOME_DIR/output/install_fails
+		echo "$PACKAGE_NAME  |  $PACKAGE_VERSION | master | $OS_NAME | GitHub | Fail |  Install_Fails" > $HOME_DIR/output/version_tracker
```
Comment (amitsadaphule) [inline s/swagger-parser/swagger-parser_ubi8.4.sh:42]: `master` seems out of place here.

File: s/swagger-parser/swagger-parser_ubi8.4.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+		echo "------------------$PACKAGE_NAME:install_&_test_both_success-------------------------"
+		echo "$PACKAGE_URL $PACKAGE_NAME" > $HOME_DIR/output/test_success 
+		echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub  | Pass |  Both_Install_and_Test_Success" > $HOME_DIR/output/version_tracker
+		find -name *.jar >> $HOME_DIR/output/post_build_jars.txt
+		echo "------------PATH of .JAR created for $WORK_DIR can be checked in text file:$HOME_DIR/output/post_build_jars.txt -----------"
+		exit 0
+	fi
+}
+
+try_mvn
```
Comment (amitsadaphule) [inline s/swagger-parser/swagger-parser_ubi8.4.sh:61]: Does the script work as is without setting `JAVA_HOME` env?

**Review by amitsadaphule — CHANGES_REQUESTED** [review]

File: s/swagger-parser/swagger-parser_ubi8.4.sh (hunk 1 above)
Comment (sapnashukla28) [reply s/swagger-parser/swagger-parser_ubi8.4.sh]: Removed master from echo statement

File: s/swagger-parser/swagger-parser_ubi8.4.sh (hunk 2 above)
Comment (sapnashukla28) [reply s/swagger-parser/swagger-parser_ubi8.4.sh:61]: Yes,

Output of running the script on a new docker:
```

[INFO] Results:
[INFO]
[INFO] Tests run: 25, Failures: 0, Errors: 0, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] swagger-parser-project 2.0.28 ...................... SUCCESS [  1.069 s]
[INFO] swagger-parser ..................................... SUCCESS [  0.353 s]
[INFO] swagger-parser-v3 .................................. SUCCESS [ 18.456 s]
[INFO] swagger-parser-v2-converter ........................ SUCCESS [  5.383 s]
[INFO] swagger-parser 2.0.28 .............................. SUCCESS [  9.436 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 36.041 s
[INFO] Finished at: 2021-12-21T08:40:34Z
[INFO] ------------------------------------------------------------------------
------------------swagger-parser:install_&_test_both_success-------------------------
------------PATH of .JAR created for /home/swagger-parser can be checked in text file:/home/output/post_build_jars.txt -----------
```

**Comment by sapnashukla28** [comment]
Hi,

I have removed master from echo statement.
Also attached the output of running the script as it is without the JAVA_HOME env. The script is working fine.
Kindly re-review.

Thanks,
Sapna.

**Pull request merged**